roles/pulp3/handlers/main.yml

```diff
@@ -1,32 +1,15 @@
 ---
 
```

Comment:
I disagree with deleting the specific systemd and service reload/restart. This is triggered on a per item basis, so the service is only reloaded and restarted when something has changed. This creates idempotency. Making a generic systemd reload is not sufficient for our purposes.

Reply:
See my comments for 'roles/pulp3/tasks/service.yml'